API/index.js

```diff
@@ -51,33 +56,59 @@ class API {
     this.cloudRecording = new CloudRecording({ api: this });
     this.meeting = new Meeting({ api: this });
     this.user = new User({ api: this });
+
+    // Unless specified, use throttle rules from ZOOM_THROTTLE_LIMIT_RULES.js
+    if (!dontUseDefaultThrottleRules) {
+      THROTTLE_LIMIT_RULES.forEach((rule) => { this._addRule(rule); });
+    }
   }
 
   /**
-   * Add a throttle rule
+   * Add a throttle rule. Only one of maxRequestsPerMinute and maxRequestsPerDay
+   *   may be included in the options.
    * @author Grace Whitney
    * @param {object} opts - object containing all options
    * @param {string} opts.template - endpoint URL template where placeholders
    *   are surrounded by curly braces. Example: /users/{userId}/meetings
    * @param {string} opts.method - method of the endpoint
-   * @param {number} [opts.maxRequestsPerInterval=unlimited] - the maximum
-   *   number of requests allowed per time interval
-   * @param {number} [opts.millisecondsPerInterval=1000] - the milliseconds per
-   *   time interval
+   * @param {number} [opts.maxRequestsPerSecond=unlimited] - the maximum number
+   *   of requests allowed per second
```

Comment:
Example: `the maximum number of requests allowed per second. Either this or opts.maxRequestsPerMinute can be defined, but not both.`